regulations/views/comment.py

```diff
@@ -11,6 +12,13 @@
 from regulations import tasks
 import requests
 
+# The following limit is specified by the regulations.gov API
+# It is not available as a queryable endpoint
```

Comment:
All I was trying to say was that it would have been nice if regulations.gov had endpoints that made these limits available dynamically as opposed to us having to _set_ it statically. I'll remove it as it is not clear.